src/setup-wp-server-files.ts

```diff
@@ -51,4 +52,5 @@ export default async function setupWPServerFiles() {
 	await copyBundledLatestWPVersion();
 	await copyBundledSqlite();
```

Comment:
I noticed that after changes introduced in https://github.com/Automattic/studio/pull/136/commits/a73fe82af9a299942b53b5c6cee4081b0ab9312d, the bundled version of SQLite has the path `wp-files/sqlite-database-integration` (i.e. no longer contains the suffix `-main`). However, in `copyBundledSqlite` function, the source path used in the copy operation references `wp-files/sqlite-database-integration-main`. This results in not copying the SQLite plugin to the Studio's server files location, but creating an empty folder. Besides, having an empty folder affects the function `downloadSqliteIntegrationPlugin`, as it prevents the download from happening ([reference](https://github.com/Automattic/studio/blob/a73fe82af9a299942b53b5c6cee4081b0ab9312d/vendor/wp-now/src/download.ts#L108-L111)).

Reply:
Maybe we could simply rename [`SQLITE_FILENAME`](https://github.com/Automattic/studio/blob/a73fe82af9a299942b53b5c6cee4081b0ab9312d/vendor/wp-now/src/constants.ts#L4) to `sqlite-database-integration`, now that the `-main` suffix won't be included since we don't download the GitHub version.

Reply:
> I noticed that after changes introduced in [a73fe82](https://github.com/Automattic/studio/commit/a73fe82af9a299942b53b5c6cee4081b0ab9312d), the bundled version of SQLite has the path `wp-files/sqlite-database-integration` (i.e. no longer contains the suffix `-main`). However, in `copyBundledSqlite` function, the source path used in the copy operation references `wp-files/sqlite-database-integration-main`. 

Thank you for testing and catching this. I'll address this oversight. 

> Maybe we could simply rename [`SQLITE_FILENAME`](https://github.com/Automattic/studio/blob/a73fe82af9a299942b53b5c6cee4081b0ab9312d/vendor/wp-now/src/constants.ts#L4) to `sqlite-database-integration`, now that the `-main` suffix won't be included since we don't download the GitHub version.

I considered this approach, but ultimately [opted to minimize](https://github.com/Automattic/studio/pull/136#discussion_r1604197174) `wp-now` changes. That said, I'm open to modifying the `SQLITE_FILENAME`. It would likely simplify the code. 👍🏻

Reply:
> I considered this approach, but ultimately [opted to minimize](https://github.com/Automattic/studio/pull/136#discussion_r1604197174) `wp-now` changes. That said, I'm open to modifying the `SQLITE_FILENAME`. It would likely simplify the code. 👍🏻

I updated locally the `SQLITE_FILENAME` constant and seems to work without further changes. @dcalhoun have you identified other parts that we might need to update in `wp-now`?

Reply:
No, I had not explored the ramifications. Thank you for testing. I will modify the value and test myself as well.